example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -49,10 +52,22 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // Set this value in your local.properties
-        if (BuildConfig.BACKTRACE_SUBMISSION_URL != null) {
-            backtraceClient = initializeBacktrace(BuildConfig.BACKTRACE_SUBMISSION_URL);
+        if (BACKTRACE_SUBMISSION_URL != null) {
+            backtraceClient = initializeBacktrace(BACKTRACE_SUBMISSION_URL);
         }
 
+        // Crash Loop Detector example
+        boolean isCLSafeModeReq = BacktraceDatabase.IsSafeModeRequired();
```

Comment:
this code breaks if crashpad is not initialized at all. We need to fix it before the final release. Not sure if this is the problem with the flow or in the crashpad internals

Reply:
Okay - so this one, following our discussions, is what comes from crashpad internals.

Reply:
We cannot accept this PR without update on the crashpad side. I'm on it.